pom.xml

```diff
@@ -569,6 +570,22 @@
 
             <!-- Keep sorted -->
 
+            <dependency>
+                <groupId>${full.maven.groupId}</groupId>
+                <artifactId>wildfly-channel</artifactId>
+                <version>${full.maven.version}</version>
+                <type>yaml</type>
+                <classifier>channel</classifier>
```

Comment:
since the channels are not attached to the reactor, I think this dependency could be removed? Or does it make sense to keep it in case we need to attache the channels in future?

Reply:
Yeah, that can be removed. When I started all this I added some random off-topic bits in case we decided to push and deploy channels as part of WF 31. But now it's right at the deadline and we need more discussion before doing that, so these can be removed.

Reply:
Fixed.